frame/im-online/src/lib.rs

```diff
@@ -225,8 +253,8 @@ decl_event!(
 
 decl_storage! {
 	trait Store for Module<T: Trait> as ImOnline {
-		/// The block number when we should gossip.
-		GossipAt get(fn gossip_at): T::BlockNumber;
+		/// The block number when we should send heartbeat.
```

Comment:
this doesn't make sense. surely there are many blocknumbers when we "should" (or rather "could") be sending a heartbeat.

i guess there is one block number in each session before which we would not expect to send a heartbeat, but there's no mention here of "after which" or "current session".